config/packages/test/content_api.yaml

```diff
@@ -0,0 +1,16 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    libero.content_store.items:
+        class: Libero\ContentApiBundle\Adapter\FilesystemItems
+        arguments:
+            - '%env(ARTICLES_PATH)%'
+
+    libero.content_store.items.schema:
+        abstract: true
```

Comment:
Just temporary since `libero.content_store.items` is `FilesystemItems` rather than `DoctrineItems` here.